include/translatable-object.php

```diff
@@ -418,7 +386,7 @@ public function get_objects_with_no_lang( $limit ) {
 		$object_ids   = wp_cache_get( $cache_key, $cache_type );
 
 		if ( ! is_array( $object_ids ) ) {
-			$object_ids = $wpdb->get_col( $sql ); // PHPCS:ignore WordPress.DB.PreparedSQL.NotPrepared
+			$object_ids = $GLOBALS['wpdb']->get_col( $sql ); // PHPCS:ignore WordPress.DB.PreparedSQL.NotPrepared
```

Comment:
Why not using `global $wpdb` anymore?

Reply:
Since we need it only once, it's shorter this way, no need for `global $wpdb;` anymore.